easybuild/easyblocks/c/cp2k.py

```diff
@@ -701,13 +701,20 @@ def test_step(self):
                     break
 
             # location of do_regtest script
-            cfg_fn = "cp2k_regtest.cfg"
+            cfg_fn = 'cp2k_regtest.cfg'
+
             regtest_script = os.path.join(self.cfg['start_dir'], 'tools', 'regtesting', 'do_regtest')
-            regtest_cmd = "%s -nosvn -nobuild -config %s" % (regtest_script, cfg_fn)
+            regtest_cmd = [regtest_script, '-nobuild', '-config', cfg_fn]
+            if LooseVersion(self.version) < LooseVersion('8.0'):
+                # -nosvn option was removed in CP2K 8.1
```

Comment:
it was already removed in 7.1, but just shows a warning there, so should be fine

Reply:
It was deprecated in 7.1, yeah, but that doesn't cause any actual trouble...

Should I change the version check to `< 7.0`?

Reply:
you could, or you could change the comment to say '-nosvn is not allowed in CP2K 8.1' or something like that.
both are fine to me

Reply:
fixed in 31f917b